### PR TITLE
AC-1666 add description to forms

### PR DESCRIPTION
**Pull request by soeren-schmaljohann-2denker**

### CODE REVIEW

File: packages/apptive_grid_core/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## 0.10.0-alpha.4
 * Adjust GridField to include `key` and `schema`
+* Add `description` to forms
```
Comment (ABausG) [inline packages/apptive_grid_core/CHANGELOG.md:3]: This version is already released so please bump to `-alpha.5`

File: packages/apptive_grid_form/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ## 0.10.0-alpha.4
 * Update Core Package to 0.10.0-alpha.4 to include breaking changes to GridField
+* Add `description` to forms
```
Comment (ABausG) [inline packages/apptive_grid_form/CHANGELOG.md:3]: Same here. This version is already released so please bump the version to -alpha.5

File: packages/apptive_grid_form/test/apptive_grid_form_test.dart [hunk 3]
```diff
@@ -97,6 +97,67 @@ void main() {
...
+        ),
+      );
+
+      await tester.pumpWidget(target);
+      await tester.pumpAndSettle();
+
+      expect(find.text('Form Description'), findsOneWidget);
+    });
+
+    testWidgets('Description do not displays', (tester) async {
```
Comment (ABausG) [inline packages/apptive_grid_form/test/apptive_grid_form_test.dart:130]: Type
```suggestion
    testWidgets('Description does not display', (tester) async {
```

File: packages/apptive_grid_form/test/apptive_grid_form_test.dart [hunk 4]
```diff
@@ -97,6 +97,67 @@ void main() {
...
+          components: [],
+          schema: {},
+          links: {},
+        ),
+      );
+
+      await tester.pumpWidget(target);
+      await tester.pumpAndSettle();
+
+      expect(find.text('Form Title'), findsNothing);
```
Comment (ABausG) [inline packages/apptive_grid_form/test/apptive_grid_form_test.dart:157]: Should check for `'Form Description'`

```suggestion
      expect(find.text('Form Description'), findsNothing);
```

**Review by ABausG — CHANGES_REQUESTED** [review]

File: packages/apptive_grid_form/CHANGELOG.md (hunk 2 above)
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_form/CHANGELOG.md]: done

File: packages/apptive_grid_core/CHANGELOG.md (hunk 1 above)
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_core/CHANGELOG.md]: done

File: packages/apptive_grid_form/test/apptive_grid_form_test.dart (hunk 4 above)
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_form/test/apptive_grid_form_test.dart:157]: done

File: packages/apptive_grid_form/test/apptive_grid_form_test.dart (hunk 3 above)
Comment (soeren-schmaljohann-2denker) [reply packages/apptive_grid_form/test/apptive_grid_form_test.dart]: done

**Review by ABausG — APPROVED** [review]

**Pull request merged**